mmv1/products/vertexai/terraform.yaml

```
@@ -62,6 +62,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: 'force_destroy'
         description: 'If set to true, any EntityTypes and Features for this Featurestore will also be deleted'
         default_value: false
+  Index: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: false
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 60
+      update_minutes: 60
+      delete_minutes: 60
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "vertex_ai_index"
+        primary_resource_id: "index"
+        vars:
+          display_name: "test-index"
+          bucket_name: "vertex-ai-index-test"
+        test_env_vars:
+          project: :PROJECT_NAME
+        # Skip the import test in an autogenerated until projects.locations.endpoints.get
+        # responds metadata.contents_delta_uri. Instead, add a manually written test.
+        skip_import_test: true
```

Rather than `skip_import_test` you can use `ignore_read_extra` to skip specific fields but still autogenerate the test:

```suggestion
        ignore_read_extra:
          - metadata.0.contents_delta_uri
```

---

I used `ignore_read_extra` in [dae899c](https://github.com/GoogleCloudPlatform/magic-modules/pull/6728/commits/dae899c350a549cfcd48dc414b4ac5684beba65d). Thanks!